code/modules/research/designs/misc_designs.dm

```diff
@@ -334,6 +334,76 @@
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_SECURITY | DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/dice_bag
+	name = "Bag of Dice"
+	desc = "An assorted bag of colorful plastic dice."
+	id = "dice_bag"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/storage/dice
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/card_deck
+	name = "Deck of Cards"
+	desc = "A deck of NT standard playing cards."
+	id = "card_deck"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/toy/cards/deck
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/crayon_box
+	name = "Box of Crayons"
+	desc = "A box full of crayons"
+	id = "crayon_box"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/plastic = 1000)
+	build_path = /obj/item/storage/crayons
```

Comment:
Why would plastic get you crayons 🤨

Reply:
(make it require colourfull reagent i know you can put chemicals in the techlathe)

Reply:
yeah thats far, only instance of chems used i know is the flomatray spray(botanist plant mutator gun) thought colourfull agent could just make the crayons for you like it isn't that hard of recipe to make you could also just make it require a low amount of it like 7 colourfull reagent for 1 box 1 colourfull reagent unit for each crayon

Reply:
sorry for the late response, i was a bit busy over the weekend. As far as I'm aware there aren't any other recipes in the protolathe the require chems currently. From what I've heard it used to be used for circuit printing, but not anymore. To be honest I agree with the decision to not require chems for lathe recipes, since it makes crafting the item a bit unintuitive and complicated. Regardless, I don't think having only one tech locked recipe require chems is a good idea. If using chems with protolathes is going to be a thing, it probably needs to be more prevalent. Since I added cardboard as a lathe mat, I decided to replace half of the recipe with cardboard, since your also printing a box. Cuts the plastic cost down a bit and adds another recipe requiring cardboard, which is good imo.